tsconfig.json

```diff
@@ -1,4 +1,3 @@
-// -*- jsonc -*-
```

Comment:
at least this one should stay, tsc is able to handle comments in the tsconfig file (but github by default is not)

Reply:
it just seemed overkill since with our imported config I've never seen a library tsconfig longer than about 5 lines